js/actions/dashboard.js

```diff
@@ -1,5 +1,10 @@
 export const SET_ACTIVITY_EXPANDED = 'SET_ACTIVITY_EXPANDED'
 export const SET_STUDENT_EXPANDED = 'SET_STUDENT_EXPANDED'
+export const SET_STUDENT_SORT = 'SET_STUDENT_SORT'
+
+export const SORT_BY_NAME = 'NAME'
+export const SORT_BY_MOST_PROGRESS = 'MOST_PROGRESS'
+export const SORT_BY_LEAST_PROGRESS = 'LEAST_PROGRESS'
```

Comment:
I added the sort constants to the actions...but maybe they're better off elsewhere? I notice that there are imports like `import * as types from '../../js/actions/dashboard'` and adding non-type constants sort of messes that up.

Reply:
Seems fine for me since we need to share that between component, action, and tests. This seems like the best place.